requirements.txt

```diff
@@ -4,6 +4,7 @@ antlr4-python3-runtime
 requests
 six
 networkx >= 2.0.0
+scipy
```

Comment:
I'm not convinced we need to add this dependency just to tensor up some Hadamards

Reply:
- [x] todo